packages/driver/src/dom/elements/nativeProps.ts

```diff
@@ -207,7 +207,6 @@ const nativeGetters = {
   body: descriptor('Document', 'body').get,
```

Comment:
This file's changes involve undoing https://github.com/cypress-io/cypress/pull/27860